tests/test_generator.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+# ! /usr/bin/env python
+""" main test script to test the primary functions/classes/methods of the medigan module.
+
+.. codeauthor:: Richard Osuala <richard.osuala@gmail.com>
```

Comment:
maybe add yourself here ;)

Reply:
I propose to remove all the header information except of the description. Code author information is in the git history.
Encoding and shebang also seem to be not really necessary. Is there any reason for having them in our project?